templatesOas/apis.mustache

```diff
@@ -22,6 +22,8 @@ import type {
 {{#hasEnums}}
 {{#operations}}
 {{#operation}}
+{{#vendorExtensions}}
```

Comment:
Essentially this PR includes/generates type and operations if
1. there are no vendor extensions
2. there are vendor extensions but no x-scapi-internal extension. Is that correct?

Reply:
Yes, that is correct